sunbird_service_api_test/env

```diff
@@ -15,4 +15,6 @@
 | sunbird_es_index_type      | Need to discuss
 | sunbird_es_index           | Elastic search index name.
 | sunbird_test_base_url      | https://dev.open-sunbird.org 
-| sunbird_default_channel    | sunbird-dev
\ No newline at end of file
+| sunbird_default_channel    | sunbird-dev
```

Comment:
Default channel value for Sunbird. This is the default channel value of a root organisation created. e.g. sunbird-dev